.pre-commit-config.yaml

```diff
@@ -591,10 +591,11 @@ repos:
         require_serial: true
       - id: update-spelling-wordlist-to-be-sorted
         name: Sort alphabetically and uniquify spelling_wordlist.txt
-        entry: ./scripts/ci/pre_commit/pre_commit_sort_spelling_wordlist.sh
-        language: system
+        entry: ./scripts/ci/pre_commit/pre_commit_sort_spelling_wordlist.py
```

Comment:
Should we remove `./scripts/ci/pre_commit/pre_commit_sort_spelling_wordlist.sh` file?

Reply:
@potiuk can you check if again?

Reply:
Now removed for sure :)